### DISCUSSION TITLE
Problem to start the Docker - newbies question

### DISCUSSION THREAD

**Discussion by Esmax666** [Second-Hand-Friends/kleinanzeigen-bot#341] [Q&A]
Hi i tried the installation with "Installation using Docker"

I installed the Docker / bash and X11 - X windows system

Question 1:
Is that normal if I start "VcXsrv Server" I have a big Black Screen ? 

Question 2:
In the Guide there is "Run the docker image:" 
I downloaded the full zip github project, but in "C:\****\kleinanzeigen-bot-main\docker\image" but under "docker" -> "builds" h want a "Dockerbuild" files
What is wrong ?

Question 3:
is that not possible to download it from internet with "dockerdesktop" -> " Images" -> search image for run ?

Question :
Where should i change / insert this code?
```
X11_DISPLAY=192.168.50.34:0.0 # replace with IP address of workstation where X11 server is running

DATA_DIR=/var/opt/data/kleinanzeigen-bot # path to config

# /mnt/data is the container's default working directory
docker run --rm --interactive --tty \
  --shm-size=256m \
  -e DISPLAY=$X11_DISPLAY \
  -v $DATA_DIR:/mnt/data \
  second-hand-friends/kleinanzeigen-bot \
  --help
  ```
  
  thx a lot

**Comment by sebthom** [comment]
> Question 1: Is that normal if I start "VcXsrv Server" I have a big Black Screen ?
yes
 
> Question 2: In the Guide there is "Run the docker image:" I downloaded the full zip github project, but in "C:****\kleinanzeigen-bot-main\docker\image" but under "docker" -> "builds" h want a "Dockerbuild" files What is wrong ?

You don't have to download anything. Also the guide doesn't say so.

> Question 3: is that not possible to download it from internet with "dockerdesktop" -> " Images" -> search image for run ?

No idea. I am not using docker desktop. 

> Question : Where should i change / insert this code?

> 
> ```
> X11_DISPLAY=192.168.50.34:0.0 # replace with IP address of workstation where X11 server is running
> 
> DATA_DIR=/var/opt/data/kleinanzeigen-bot # path to config
> 
> # /mnt/data is the container's default working directory
> docker run --rm --interactive --tty \
>   --shm-size=256m \
>   -e DISPLAY=$X11_DISPLAY \
>   -v $DATA_DIR:/mnt/data \
>   second-hand-friends/kleinanzeigen-bot \
>   --help
> ```
> 
> thx a lot

To commands are supposed to be executed in Bash.

**Comment by sebthom** [comment]
Also, the docker image name is not `second-hand-friends/kleinanzeigen-bot` but `ghcr.io/second-hand-friends/kleinanzeigen-bot`
